src/apphosting/config.ts

```diff
@@ -39,12 +40,28 @@ export interface Config {
 }
 
 /**
- * Finds the project root for apphosting backends.
- * Starts with cwd and walks up the path until an apphosting.yaml file is found
+ * Returns the absolute path for an app hosting backend root.
  *
- * Example path that's returned: "/home/my-project"
+ * If backendRootRelativeToProjectRoot is provided then this function will
+ * try to return the absolute path of the backend root
```

Comment:
small nit: I think backendRootRelativeToProjectRoot should be it's own function, its functionality is completely distinct from when it's not passed in